run_genscan.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+"""
+Accessing http://hollywood.mit.edu/GENSCAN.html to obtain exons and introns 
+from a nucleotide sequence. 
+"""
+
+from bs4 import BeautifulSoup
+import requests
+import re
+import pandas as pd
+from typing import IO, Union, Literal
+from dataclasses import dataclass
+
+@dataclass
+class GenscanOutput:
+    status: str
+    cds_list: str
+    intron_list: pd.DataFrame
+    exon_list: pd.DataFrame
+
+def run_genscan(sequence: str = None,
+                sequence_file: Union[str, IO] = None,
+                organism: Literal["Vertebrate", "Arabidopsis", "Maze"] = "Vertebrate",
+                exon_cutoff: float = 1.00,
+                sequence_name: str = "") -> GenscanOutput:
+                
+        FORM_URL = "http://hollywood.mit.edu/cgi-bin/genscanw_py.cgi"
+        
+        # Choose if the form is to be filled using a file or "manual" input
+        if sequence_file:
+            data = {"-o":organism,
+                    "-e":exon_cutoff,
+                    "-n":sequence_name,
+                    "-p":"Predicted peptides only",
+                    "-u":sequence_file}
+        else:
+            data = {"-o":organism,
+                    "-e":exon_cutoff,
+                    "-n":sequence_name,
+                    "-p":"Predicted peptides only",
+                    "-s":sequence}
+
+        response = requests.post(url=FORM_URL, data=data)
+        status = response.status_code
+        
+        soup = BeautifulSoup(response.content, "lxml")
+        text = soup.find_all("pre").__repr__()
```

Comment:
Если вместо дандера возможно использовать функцию, то стоит использовать именно её
```suggestion
        text = repr(soup.find_all("pre"))
```